src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2475,6 +2498,31 @@ namespace winrt::TerminalApp::implementation
         return _isAlwaysOnTop;
     }
 
+    bool TerminalPage::InboundPty() const
+    {
+        return _isInboundPty;
+    }
+
+
+    void TerminalPage::_OnInboundPtyChanged(const IInspectable& /*sender*/,
+                                           const IInspectable& /*eventArgs*/)
+    {
+        if (InboundPty())
+        {
+            ConptyConnection::StartInboundListener();
+        }
+        else
+        {
+            ConptyConnection::StopInboundListener();
+        }
+    }
+
+    void TerminalPage::_OnNewConnection(winrt::Microsoft::Terminal::TerminalConnection::ITerminalConnection connection)
+    {
+        // TODO: this should probably use a more reasonable profile than nullptr.
+        _OpenNewTab(nullptr, connection);
```

Comment:
`nullptr` gets evaluated as the default profile (`CascadiaSettings::BuildSettings()`-->`CascadiaSettings::_GetProfileForArgs()`). Idk what the design is to choose which profile to open this in. But I guess this is perfectly fine and future-proof for now.

Reply:
You and I (and I and Michael) were having parallel discussions about what this should be.

It should be `profiles.defaults` given form, **not** the `defaultProfile`.

Reply:
Right! It's that archetype discussion. Sounds like a TODO for later then haha

Reply:
#9458 will have the additional data provisions for us to either make a profile on the fly from the LNK/REG data or to otherwise attempt to more appropriately match this.

If `nullptr` is a sufficient default for now, we can refine it later.